modules/data-entry/src/main/frontend/src/dataHomepage/FilterComponents/SubjectFilter.jsx

```diff
@@ -57,7 +57,9 @@ const SubjectFilter = forwardRef((props, ref) => {
 
   let constructQuery = (query, requestID) => {
     let url = new URL("/query", window.location.origin);
-    let sqlquery = "SELECT s.* FROM [lfs:Subject] as s" + (query.search ? ` WHERE CONTAINS(s.'identifier', '*${query}*')` : "");
+    let formattedQuery = query.replace(/\s*\/\s*/g, " / ");
+    let sqlquery = "SELECT s.* FROM [lfs:Subject] as s" + (query.search ? ` WHERE CONTAINS(s.'fullIdentifier', '*${formattedQuery}*')` : "");
```

Comment:
Not really a bug in this PR, but the issue with `CONTAINS` is that it will process the stored value and the query, and searching for things like `1/1`, which is what real tumor identifiers look like in LFS, will fail. A temporary fix is to use this instead:

```suggestion
    let sqlquery = "SELECT s.* FROM [lfs:Subject] as s" + (query.search ? ` WHERE s.'fullIdentifier' LIKE '%${formattedQuery}%')` : "");
```

`formattedQuery` would also need to have special LIKE characters escaped.

But this is something for a different PR, since it may break other kinds of searches.